README.md

```diff
@@ -42,13 +42,13 @@ To install the current release, which includes support for
 Windows)*:
 
 ```
-$ pip install tensorflow
+$ pip3 install tensorflow
```

Comment:
Let's not change this file. In several usecases `pip` is `pip3` already.

Reply:
Sure. Then let me revert the changes